main.go

```diff
@@ -35,17 +132,19 @@ func build(diskPath string) (v1.Image, error) {
 	return image, nil
 }
 
-func push(image v1.Image, imageDestination string) error {
+func push(image v1.Image, containerDiskName string) error {
```

Comment:
I would rename it to `pushContainerDisk`

Reply:
Done.